ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -94,6 +94,7 @@ open class AvatarView: UIView {
         static let borderWidth: CGFloat = 2
```

Comment:
should there be more documentation on borderWidth vs fameborderwidth difference?